### ISSUE TITLE
feat: docker images for a provider

### ISSUE THREAD

**Issue by dopry** [jazzband/django-oauth-toolkit#1359]
Wiring up DOT in a Django instance can be time consuming. Sometimes you just want a simple standalone IDP.  I feel it would make DOT more accessible and maybe expand our community. 

**Describe the solution you'd like**
A docker image based on the tests/app/idp

The templates folder should be exposed as a VOLUME.
The setting should use Django-Environ so the instance can be configured using ENV vars passed to the container.

**Describe alternatives you've considered**
Keycloak - not as easy for me as django/python developer to eject from and customize.
Ory - you need 3 services with their platform (Kratos, Hydra, OathKeeper) and it doesn't really have a UI, DOT + Django wraps up these 3 roles in one service and provides an easy admin interface.

**Additional context**
Not sure if this would belong with DOT but it would be nice to have it publish in parallel with the same versioning as a sort of 'dogfood/canary' release.

**Comment by dopry** [comment]
@n2ygk: I was wondering what you thnk of this idea?

**Comment by n2ygk** [comment]
@dopry sounds like a good idea.
